packages/next/server/next-dev-server.js

```diff
@@ -272,11 +276,17 @@ export default class DevServer extends Server {
 
     // In dev mode we use on demand entries to compile the page before rendering
     try {
-      await this.hotReloader.ensurePage(pathname).catch(err => {
+      await this.hotReloader.ensurePage(pathname).catch(async err => {
         if (err.code !== 'ENOENT') {
           return Promise.reject(err)
```

Comment:
if the function is now `async`, this could be `throw err`, right?

Reply:
Updated